.argo-ci/ci.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     - name: revision
       value: master
     - name: repo
-      value: https://github.com/CyrusBiotechnology/argo.git
+      value: https://github.com/cyrusbiotechnology/argo.git
```

Comment:
did you hit a bug related to casing?

Reply:
Yep. go's codegen tool insists on generating code with lowercase imports, and if you have a mix of lowercase and mixed case imports in your source base, your code wont compile with an error about import case collisions. it's "very cool"